prow/plugins/approve/approvers/owners.go

```diff
@@ -169,7 +173,7 @@ func (o Owners) GetOwnersSet() sets.String {
 	return owners
 }
 
-// Shuffles the potential approvers so that we don't always suggest the same people
+// GetShuffledApprovers : Shuffles the potential approvers so that we don't always suggest the same people
```

Comment:
The format is `// method shuffles blah blah blah`

AKA
* The first word of the sentence is the method name. 
* There's no colon
* the second word of the sentence is lowercase.